packages/swapper/src/swappers/zrx/utils/helpers/helpers.ts

```diff
@@ -69,56 +68,75 @@ export const getAllowanceRequired = async ({
   web3,
   erc20AllowanceAbi
 }: GetAllowanceRequiredArgs): Promise<BigNumber> => {
-  if (sellAsset.symbol === 'ETH') {
-    return bn(0)
-  }
+  try {
+    if (sellAsset.assetId === 'eip155:1/slip44:60') {
+      return bn(0)
+    }
 
-  const ownerAddress = receiveAddress
-  const spenderAddress = allowanceContract
-  const tokenId = sellAsset.tokenId
+    const ownerAddress = receiveAddress
+    const spenderAddress = allowanceContract
+    const tokenId = sellAsset.tokenId
```

Comment:
similarly, don't use this

Reply:
https://github.com/shapeshift/lib/issues/653